Gemfile

```diff
@@ -18,7 +18,6 @@ gem "audited"
 gem "aws-sdk-cloudfront"
 gem "aws-sdk-s3"
 gem "aws-sdk-waf"
-gem "biodiversity"
```

Comment:
This just kicks the can down the road. Either refactor https://github.com/inaturalist/inaturalist/blob/main/lib/ratatosk/lib/ratatosk/name_providers/eol.rb#L215 or remove the EOL name provider since I don't think we use it any more and don't have plans to bring it back. My preference is for the latter.

Reply:
I removed the EOL name provider